backend/src/rest/assignments.js

```diff
@@ -14,8 +14,12 @@ router.post('/', async (req, res) => {
   let title = req.body.title
   let description = req.body.description
   let contact = req.body.contact
+  let customerFee = req.body.customerFee
+  let customerCompanyURL = req.body.customerCompanyURL
+  let clientHourlyRate = req.body.clientHourlyRate
+  let location = req.body.location
 
-  if (typeof customerName !== 'string' || typeof title !== 'string' || typeof description !== 'string' || typeof contact !== 'string') {
+  if (typeof customerName !== 'string' || typeof title !== 'string' || typeof description !== 'string' || typeof contact !== 'string' || typeof contact !== 'string') {
```

Comment:
Angående @pola s kommentar om validering, är det `location` som egentligen ska vara här? Annars valideras contact 2 ggr.
```suggestion
  if (typeof customerName !== 'string' || typeof title !== 'string' || typeof description !== 'string' || typeof contact !== 'string' || typeof location !== 'string') {
```

Reply:
Tror det är resten av parametrarna som saknades. Jag löser det :) Och yes, location skall med!